src/components/BooksQuery.tsx

```diff
@@ -1,45 +1,42 @@
-import axios from "axios";
 import { useState } from "react";
 import Card from "@mui/material/Card";
 import "./books.css";
 import { CardContent, Typography } from "@mui/material";
```

Comment:
Please do not use a third party React styling framework like MaterialUI. Do your own stylings